### PR TITLE
Add flag for debounce delay

### LINKED ISSUE #31: Request: add CLI option to set debounce delay value
It would be nice to set the debounce delay value through a CLI option.
100ms is hard-coded today, but some set ups may require longer values.

### PR DESCRIPTION
**Pull request by dougkoch**
Fixes #31

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by dougkoch** [comment]
Jenkins, test this please

**Comment by achew22** [comment]
Test this

**Comment by achew22** [comment]
Test this please

File: ibazel/main.go [hunk 1]
```diff
@@ -19,12 +19,14 @@ import (
...
 	"os"
 	"strings"
+	"time"
 )
 
 var overrideableBazelFlags []string = []string{
 	"--test_output=",
 }
 
+var debounceDuration = flag.Int("debounce", 100, "Debounce duration in milliseconds")
```
Comment (achew22) [inline ibazel/main.go]: What would you think about using a duration flag?

https://golang.org/pkg/flag/#Duration

Comment (dougkoch) [reply ibazel/main.go]: Done. Good catch

**Pull request merged**